File: cpp/bc1178.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	double vec[100], val;
	
	cin >> val;
	
	vec[0] = val;
	
	for(int i = 1; i < 100; i++)
		vec[i] = vec[i-1] / 2.0;
	for(int i = 0; i < 100; i++)
		cout << "N[" << i << "] = " << fixed << setprecision(4) << vec[i] << endl; 
}

File: python/bc1178.py
val = float(input())
vec = [None]*100
vec[0] = val

for i in range(1, 100):
    vec[i] = vec[i-1] / 2.0

for c in range(0, 100):
    print(f"N[{c}] = {vec[c]:.4f}")